src/users/users.service.ts

```diff
@@ -13,12 +13,13 @@ import { User, UserDocument } from 'src/shared/schema';
 import { BcryptUtil, CloudinaryFolders, firstCapitalize, getMailTemplate, getPaginated, getPagingParams, passwordMatch, sendMail, uploadToCloudinary, verifyHandle } from 'src/shared/utils';
 import { ApiReq, EmailFromType, UserRole, UserStatus } from 'src/shared/interfaces';
 import { UserInviteDto } from './dto/user-invite.dto';
+import * as sizeOf from 'image-size';
```

Comment:
-- Suggestion -- 
This package is not used,  and this body of work will be fine without the package. 

It will be great to have this removed so we can close this out and not add package which we will not use in the application